debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -885,6 +886,27 @@ public static final Field MASK_COLUMN(int length) {
                         + "be masked with " + length + " asterisk ('*') characters.");
     }
 
+    /**
+     * Method that generates a Field for specifying that string columns whose names match a set of regular expressions should
+     * have their values masked by the {@param hashAlgorithm} and {@param salt}.
+     *
+     * @param salt the salt that is used within the hash function
+     * @param hashAlgorithm the hash function that is used to mask the columns string values written in source records;
+     *                      must be on of Java Cryptography Architecture Standard Algorithm {@link MessageDigest}.
+     * @return the field; never null
+     */
+    public static final Field MASK_COLUMN_WITH_HASH(String salt, String hashAlgorithm) {
```

Comment:
This should be added to `configDef()` and `ALL_FIELDS`. The definition should go into `RelationalDatabaseConnectorConfig`, so it can be used for all the relational connectors.

Reply:
For a general solution that doesn't copy code, I'm not sure how to handle the static generator function.
I could just move `MASK_COLUMN_WITH_HASH` to `RelationalDatabaseConnectorConfig`. 

At the moment I don't see any field generated by such function added to `configDef()` or `ALL_FIELDS`.